makefile

```diff
@@ -13,7 +13,7 @@ all: build-lib
 
 .PHONY: build-lib
 build-lib:
-	clang -E -P source/ctod/c_parser.c > source/ctod/c_parser_pre.c
+	clang -E -Isource -P source/ctod/c_parser.c > source/ctod/c_parser_pre.c
```

Comment:
I forgot I even had this messy makefile here, this was supposed to preprocess for ImportC before it could do that itself.